jiraworklog/src/Component/User/User.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+import { Button, Modal } from "semantic-ui-react";
+import moment from "moment";
+import "./User.css";
+
+const User = (props) => {
+
+  const [show, setShow] = useState(false);
+  const handleClose = () => setShow(false);
+  const handleShow = () => setShow(true);
+
+  return (
+    <>
```

Comment:
use destructuring to grab necessary items .. for example 
`const { name, comments, time } = props;`